raster3d/r3.stats/support.c

```diff
@@ -271,10 +271,10 @@ void print_stat_table(stat_table *stats, int counts_only)
     }
     else if (counts_only) {
 	for (i = 0; i < stats->nsteps; i++) {
-	    fprintf(stdout, "%d %ld\n",
+	    fprintf(stdout, "%d %i\n",
```

Comment:
Why is `i` better than `d`? According to `man fprintf`, `i` and `d` are identical.

Reply:
No particular reason I used `i`, will change to `d` as it is in general use in existing code.